MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -817,16 +819,10 @@ public void actionPerformed(ActionEvent action) {
                 }
                 break;
             }
-            case CMD_TRYING_TO_MARRY: {
-                if (people.length > 1) {
-                    boolean status = !people[0].isTryingToMarry();
-                    for (Person person : people) {
-                        person.setTryingToMarry(status);
-                        gui.getCampaign().personUpdated(person);
-                    }
-                } else {
-                    selectedPerson.setTryingToMarry(!selectedPerson.isTryingToMarry());
-                    gui.getCampaign().personUpdated(selectedPerson);
+            case CMD_MARRIAGEABLE: {
+                final boolean marriageable = !people[0].isMarriageable();
+                for (final Person person : people) {
+                    person.setMarriageable(marriageable);
```

Comment:
Just checking: The personUpdated is unnecessary?

Reply:
Yeah, the flag is entirely internal.